editor/components/inserter/style.scss

```diff
@@ -27,6 +27,10 @@
 	&:hover {
 		color: #00aadc;
 	}
+
+	& .insert {
```

Comment:
- We can drop the `&` and the nesting would work just as well.
- I had a hard time finding what `.insert` was in this case; found later that it's the Dashicon. For future readers, maybe being overly specific could clarify, e.g. `.dashicon.insert`